# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../zaptel/split_make_install.patch
# Copyright (C) 2006 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

Note: Just build and install userspace stuff of zaptel 
2006-04-02 Christian Wiese

--- zaptel-1.2.5/Makefile.orig	2006-04-02 22:26:04.000000000 +0200
+++ zaptel-1.2.5/Makefile	2006-04-02 22:50:37.000000000 +0200
@@ -153,6 +153,8 @@
 
 all: $(BUILDVER) $(LIBTONEZONE_SO)
 
+bins: $(BINS)
+
 linux24: prereq $(MODULESO) $(BINS)
 
 linux26: prereq $(BINS)
@@ -346,14 +348,7 @@
   endif
 endif
 
-install: all devices
-	install -D -m 755 ztcfg $(INSTALL_PREFIX)/sbin/ztcfg
-	if [ -f sethdlc-new ]; then \
-		install -D -m 755 sethdlc-new $(INSTALL_PREFIX)/sbin/sethdlc; \
-	elif [ -f sethdlc ]; then \
-		install -D -m 755 sethdlc $(INSTALL_PREFIX)/sbin/sethdlc ; \
-	fi
-	if [ -f zttool ]; then install -D -m 755 zttool $(INSTALL_PREFIX)/sbin/zttool; fi
+install_modules:
 	if [ -f zaptel.ko ]; then \
 		for x in $(MODULESKO); do \
 			rm -f $(INSTALL_PREFIX)/lib/modules/$(KVERS)/extra/$$x ; \
@@ -372,6 +367,15 @@
 		fi; \
 		rm -f $(INSTALL_PREFIX)/lib/modules/$(KVERS)/misc/wcfxs.o; \
 	fi
+
+install:
+	install -D -m 755 ztcfg $(INSTALL_PREFIX)/sbin/ztcfg
+	if [ -f sethdlc-new ]; then \
+		install -D -m 755 sethdlc-new $(INSTALL_PREFIX)/sbin/sethdlc; \
+	elif [ -f sethdlc ]; then \
+		install -D -m 755 sethdlc $(INSTALL_PREFIX)/sbin/sethdlc ; \
+	fi
+	if [ -f zttool ]; then install -D -m 755 zttool $(INSTALL_PREFIX)/sbin/zttool; fi
 	install -D -m 755 $(LIBTONEZONE_SO) $(INSTALL_PREFIX)/usr/lib/$(LIBTONEZONE_SO).$(LIBTONEZONE_SO_MAJOR_VER).$(LIBTONEZONE_SO_MINOR_VER)
 	[ `id -u` = 0 ] && /sbin/ldconfig || :
 	rm -f $(INSTALL_PREFIX)/usr/lib/$(LIBTONEZONE_SO)
@@ -385,7 +389,7 @@
 	install -D -m 644 tonezone.h $(INSTALL_PREFIX)/usr/include/tonezone.h
 	install -m 644 doc/ztcfg.8 $(INSTALL_PREFIX)/usr/share/man/man8
 	install -m 644 doc/zttool.8 $(INSTALL_PREFIX)/usr/share/man/man8
-	if [ -n "$(MODCONF)" ]; then \
+	if false; then \
 		if [ -f $(MODCONF) ]; then mv -f $(MODCONF) $(MODCONF).bak ; fi; \
 		cat $(MODCONF).bak | grep -v "alias char-major-250" | \
 		grep -v "post-install torisa /sbin/ztcfg" | \
